ext/drmgr/drmgr.h

```diff
@@ -165,6 +165,19 @@ typedef dr_emit_flags_t (*drmgr_insertion_cb_t)(void *drcontext, void *tag,
                                                 bool for_trace, bool translating,
                                                 void *user_data);
 
+/**
+ * Callback function for opcode based instrumentation. In particular, this callback
+ * is triggered only for specific instruction opcodes. This is done during the
+ * third stage, i.e., instrumentation insertion.
```

Comment:
nit: you s/instrumentation insertion/ right above but then used the phrase here

Reply:
Unlike cbs for instr insert instrumentation, opcode cbs are not paired with any prior analysis cbs for now. Shall I add the parameter and pass NULL?

Reply:
Done - reverted back (looked odd with two nouns, but it is pretty much used everywhere in the header file).